gradle/plugins/src/main/kotlin/com.hedera.gradle.protobuf.gradle.kts

```diff
@@ -23,31 +23,23 @@ plugins {
 
 // Configure Protobuf Plugin to download protoc executable rather than using local installed version
 protobuf {
-    val libs = the<VersionCatalogsExtension>().named("libs")
-    protoc { artifact = "com.google.protobuf:protoc:" + libs.findVersion("google-proto").get() }
+    protoc { artifact = "com.google.protobuf:protoc" }
     plugins {
         // Add GRPC plugin as we need to generate GRPC services
-        id("grpc") {
-            artifact = "io.grpc:protoc-gen-grpc-java:" + libs.findVersion("grpc-proto").get()
-        }
+        id("grpc") { artifact = "io.grpc:protoc-gen-grpc-java" }
     }
     generateProtoTasks {
         all().configureEach { plugins { id("grpc") { option("@generated=omit") } } }
     }
 }
 
-sourceSets.all {
-    val compileProtoPath = getTaskName("", "compileProtoPath")
-    dependencies {
-        // For dependencies of protobuf compilation use versions from 'hedera-dependency-versions',
-        // but not 'runtime' dependencies of the platform (JAVA_API instead of JAVA_RUNTIME).
-        dependencies {
-            compileProtoPath(project(":hedera-dependency-versions")) {
-                attributes {
-                    attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage.JAVA_API))
-                    attribute(Category.CATEGORY_ATTRIBUTE, objects.named(Category.REGULAR_PLATFORM))
-                }
-            }
+configurations.configureEach {
```

Comment:
Need to do some trickery here due to how the Protobuf Gradle Plugin searches for dependencies to the "Tools" (Protobuf Compiler) it uses. This will go into the separate Gradle Config repo, in the next step, when we move this over.